### ISSUE TITLE
about model result

### ISSUE THREAD

**Issue by LiUzHiAn** [DevKiHyun/RDN-Tensorflow#3]
Hi,

Can I know your test result based on the model you have trained ? Specificlly, what is the PSNR value on Set5 test set (or other benchmark dataset ) of x2 scale .

**Comment by DevKiHyun** [comment]
Hi @LiUzHiAn . I didn't upload psnr result because my rdn model is different to paper's result. In calculating psnr, rdn paper uses matlab and I use opencv of python. But don't worry. RDN performance is really good about other super resolution model. I will upload complete result soon.
